aptos-move/aptos-gas/src/transaction/mod.rs

```diff
@@ -113,6 +113,11 @@ crate::params::define_gas_parameters!(
             { 5.. => "free_write_bytes_quota" },
             1024, // 1KB free per state write
         ],
+        [
+            free_event_bytes_quota: NumBytes,
+            { 7.. => "free_event_bytes_quota" },
+            1024, // 1KB free event bytes per transaction
```

Comment:
Are you planning to benchmark 2KB after v1.4 cut? This is still fine actually as it'll be another week or two before we upgrade the gas schedule in testnet

Reply:
the 2KB is for the state write quota (`free_write_bytes_quota`), orthogonal to this.